src/pkg/packaging/deb/package.targets

```diff
@@ -4,12 +4,19 @@
 
   <UsingTask TaskName="ReplaceFileContents" AssemblyFile="$(LocalBuildToolsTaskDir)core-setup.tasks.dll"/>
 
+  <PropertyGroup>
+    <IsDebianBasedDistro Condition="$(PackageTargetRid.StartsWith('ubuntu')) or
```

Comment:
I guess this will work. We have issue https://github.com/dotnet/core-setup/issues/2252 tracking the cleanup of these distro rids.